package.json

```diff
@@ -129,14 +138,19 @@
           "group": "inline@2"
         },
         {
-          "command": "tfvc.ResolveTakeTheirs",
+          "command": "tfvc.Undo",
           "when": "scmProvider == tfvc && scmResourceGroup == conflicts",
           "group": "inline@1"
```

Comment:
Yeah, as I was playing with conflicts, I came to the same conclusion - that we needed Undo here.